src/App.tsx

```diff
@@ -1,13 +1,61 @@
 import React, { useRef } from 'react';
 import './App.css';
+import styled from 'styled-components';
 
 import EmailEditor, { ToolConfig } from 'react-email-editor';
 
+const Bar = styled.div`
+  flex: 1;
+  background-color: #fcc338;
+  color: #000;
+  padding: 10px;
+  display: flex;
+  max-height: 40px;
+  h1 {
+    flex: 1;
+    font-size: 16px;
+    text-align: left;
+  }
+  button {
+    flex: 1;
+    padding: 10px;
+    margin-left: 10px;
+    font-size: 14px;
+    font-weight: bold;
+    background-color: #000;
+    color: #fff;
+    border: 0px;
+    max-width: 150px;
+    cursor: pointer;
+  }
+`;
+
 const App: React.FC = () => {
-  const emailEditorRef = useRef(null);
+  // eslint-disable-next-line
+  const emailEditorRef: any = useRef(null);
+
+  const saveDesign = () => {
+    // eslint-disable-next-line
+    emailEditorRef.current.editor.saveDesign((design: any) => {
+      console.log('saveDesign', design);
+    });
+  };
+
+  const exportHtml = () => {
+    // eslint-disable-next-line
+    emailEditorRef.current.editor.exportHtml((data: any) => {
+      const { html } = data;
```

Comment:
Is there something else than `html` property in `data`?

Reply:
Yes, we have `amp`, `chunks`, `design `and `html`
![image](https://user-images.githubusercontent.com/45947909/130673344-5a04b37e-539f-4b41-aef1-733c0005aec3.png)

Reply:
I do not know what it is, but if you think that it could be useful in the future, I think that we could include it in the console log, to avoid forgetting that we have it (no strong opinion).

Reply:
based on your explanation, I think that using it as it is right now (with destructuring) is the best option.